providers/email_setup.rb

```diff
@@ -69,12 +69,22 @@ def setup_gecos_as_default_profile(profile_ini)
     if os_supported? &&
        (policy_active?('users_mgmt', 'email_setup_res') ||
         policy_autoreversible?('users_mgmt', 'email_setup_res'))
+ 
+      directory '/var/cache/gecos' do
```

Comment:
Probably unneeded if we use 'recursive' parameter in deeper directory

See https://docs.chef.io/resource_directory.html

Reply:
I agree. I simply moved old code out of a loop. I'll fix it.

Reply:
Fixed in PR #257